src/app-check/app-check-api-client-internal.ts

```diff
@@ -86,6 +87,30 @@ export class AppCheckApiClient {
       });
   }
 
+  public verifyReplayProtection(token: string): Promise<boolean> {
```

Comment:
@weixifan this is an internal method, but I renamed it to `verifyReplayProtection` from `verifyOneTimeUseProtection`. WDYT?

Reply:
Yeah that sounds fine to me -- in any case since this is internal it can be changed in the future too.